crawl-ref/source/ability.cc

```diff
@@ -646,6 +648,15 @@ static vector<ability_def> &_get_ability_list()
         { ABIL_WU_JIAN_WALLJUMP, "Wall Jump",
             0, 0, 0, {}, abflag::berserk_ok },
 
+        // Ignis
+        { ABIL_IGNIS_SEA_OF_FIRE, "Sea of Fire",
+            0, 0, 8, /*avg 12 uses from 150 to <30 piety*/
+            {fail_basis::invo}, abflag::quiet_fail },
+        { ABIL_IGNIS_FOXFIRE, "Foxfire Swarm",
+            0, 0, 16, {fail_basis::invo}, abflag::quiet_fail },
+        { ABIL_IGNIS_RISING_FLAME, "Rising Flame",
+            0, 0, 32, {fail_basis::invo}, abflag::none },
```

Comment:
These costs feel about right; however in the `a`bility menu they both are listed as `extremely large`. A better indication of their relative cost in that menu would be nice, so that it's not only stuck in the second pane of `^`.